### ISSUE TITLE
Attempt at Angular 5.x + .NET Core 2.x migration

### ISSUE THREAD

**Issue by unquietwiki** [unosquare/passcore#134]
In #124 , I mentioned trying to fix it myself; and ended up building the project as a .NET Core 2.x application. However, I wasn't getting all the right forms to appear. So then I dug into the Typescript / Angular code (which honestly, I'm not an expert at), and attempted to migrate that code to enable Angular 5.x + AOT support. I also saw some changes on this codebase to deprecate SWAN / LDAP, so tried to add that code too. I created a "src/references.md" file as to what I used for the Angular stuff; any other required documentation can go there too.

https://github.com/unquietwiki/passcore

**Comment by Israel-94** [comment]
@unquietwiki so, what's your plan? Will you make a pull request for this version of passcore?

**Comment by unquietwiki** [comment]
@Israel-94 I'd like to: but not if it's not working! 😆

So I know from the attempted .NET changeover, and seeing some of the output without running IIS, that the back-end appears to take requests fine. I'm currently at a loss to figure out how to make sure it's using the Webpack-compiled Angular code though. On a side note, I picked up a lot of JS-experience the past month from working on https://github.com/unquietwiki/floorplan : Webpack was a pain in my ass, so I ended up using Gulp + Babel.  https://github.com/s-panferov/awesome-typescript-loader appears to be something in between: works with Webpack, but uses Babel to do its magic.
